Makefile

```diff
@@ -21,6 +23,9 @@ bin/kubebuilder:
 test: generate fmt vet manifests bin/kubebuilder
 	KUBEBUILDER_ASSETS="$(shell pwd)/bin/kubebuilder/bin" go test ./... -coverprofile cover.out
 
+kind: generate fmt vet manifests
+	go test ./internal/test/e2e --kind --repo-root ${ROOTDIR} -v
```

Comment:
you can use `${CURDIR}` which always holds the working directory

Reply:
Is there a way to get the makefile's directory? That's what I really want.

Reply:
theres no simple cross-platform way i could find when i was googling this yesterday 😢  might be possible though